application/classes/Ushahidi/Repository/Notification.php

```diff
@@ -0,0 +1,100 @@
+<?php defined('SYSPATH') OR die('No direct access allowed.');
+
+/**
+ * Ushahidi Notification Repository
+ *
+ * @author     Ushahidi Team <team@ushahidi.com>
+ * @package    Ushahidi\Application
+ * @copyright  2014 Ushahidi
+ * @license    https://www.gnu.org/licenses/agpl-3.0.html GNU Affero General Public License Version 3 (AGPL3)
+ */
+
+use Ushahidi\Core\Entity;
+use Ushahidi\Core\SearchData;
+use Ushahidi\Core\Entity\Notification;
+
+class Ushahidi_Repository_Notification extends Ushahidi_Repository
+{
+	protected function notificationExists(Entity $entity)
+	{
+		$result = DB::select('id')
```

Comment:
Ideally use `$this->selectQuery()` here or at least reference `this->getTable()` instead of hard coding the table name